apps/fz_http/lib/fz_http_web/templates/auth/request.html.heex

```diff
@@ -25,6 +25,7 @@
     <div class="field">
       <div class="control">
         <%= submit "Sign In", class: "button" %>
+        <%= link("Forgot password", to: Routes.auth_path(@conn, :forgot_password), class: "forgot-password") %>
```

Comment:
Instead of adding custom CSS for this, thoughts on using a[ Bulma level ](https://bulma.io/documentation/layout/level/)to left-align the Sign In and right-align the `Forgot password`?

```eex
...
<div class="control">
  <div class="level">
    <div class="level-left">
      <%= submit "Sign In", class: "button" %>
    </div>
    <div class="level-right">
      <%= link("Forgot password", to: Routes.auth_path(@conn, :forgot_password)) %>
    </div>
  </div>
</div>
```

Reply:
Thanks. Needed some guidance on this one. I'm familiar with Bootstrap and Tailwindcss. Never used Bulma before.